modules/zip/zip_reader.h

```diff
@@ -0,0 +1,60 @@
+/*************************************************************************/
+/*  zip_reader.h                                                         */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2022 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2022 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#ifndef ZIP_READER_H
+#define ZIP_READER_H
+
+#include "core/object/ref_counted.h"
+
+#include "core/io/file_access.h"
+#include "core/os/os.h"
+#include "thirdparty/minizip/unzip.h"
+
+class ZIPReader : public RefCounted {
+	GDCLASS(ZIPReader, RefCounted)
+
+	Ref<FileAccess> fa;
+	unzFile uzf;
+
+protected:
+	static void _bind_methods();
+
+public:
+	Error open(String path);
+	Error close();
+
+	PackedStringArray get_files();
+	PackedByteArray read_file(String path, bool case_sensitive);
```

Comment:
Parameters should start with `p_`:
```suggestion
	Error open(String p_path);
	Error close();

	PackedStringArray get_files();
	PackedByteArray read_file(String p_path, bool p_case_sensitive);
```
(Same in other classes and in the .cpp)

Reply:
That's a convention only for C++ code though, the names given in `bind_method` should not have the `p_` prefix, so they're fine as is.